src/Features/LanguageServer/Protocol/Handler/SourceGenerators/GetTextHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Roslyn.Utilities;
+using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Handler
+{
+    [ExportCSharpVisualBasicStatelessLspService(typeof(GetTextHandler)), Shared]
+    [Method("sourceGenerator/getText")]
+    internal class GetTextHandler : ILspServiceDocumentRequestHandler<SourceGeneratorGetTextParams, SourceGeneratedDocumentText>
```

Comment:
nit: would love if the name of this type was more like SourceGeneratorGetTextHandler.  Would be clearer based on that name alone what it was doing, instead of having to look at the method name.

Reply:
I went back and forth here but it kinda looked like other handlers kept the namespace/folder name for the first bit, a the handler name after that. But it really wasn't consistent. @dibarbet did we have some plan here? I'm happy to follow that if there is one; otherwise happy to take @CyrusNajmabadi's suggestion.

Reply:
Leaning towards `SourceGeneratorGetText`, its more inline with the actual method name.  agree it hasn't been consistent though

Reply:
Renamed.